src/org/loklak/api/search/TimeAndDateService.java

```diff
@@ -35,41 +38,82 @@
 
 public class TimeAndDateService extends AbstractAPIHandler implements APIHandler {
 
-	private static final long serialVersionUID = 6808423132726076271L;
-
-	@Override
-	public String getAPIPath() {
-		return "/api/timeanddate.json";
-	}
-
-	@Override
-	public BaseUserRole getMinimalBaseUserRole() {
-		return BaseUserRole.ANONYMOUS;
-	}
-
-	@Override
-	public JSONObject getDefaultPermissions(BaseUserRole baseUserRole) {
-		return null;
-	}
-
-	public JSONObject serviceImpl(Query call, HttpServletResponse response, Authorization rights, JSONObjectWithDefault permissions)
-			throws APIException {
-		return timeAndDate();
-	}
-
-	public static SusiThought timeAndDate() {
-		
-		JSONObject timeAndDate = new JSONObject();
-		
-		Date time_and_date = new Date();
-		timeAndDate.put("time_and_date", time_and_date.toString());
-		
-		
-		JSONArray jsonArray = new JSONArray();
-		jsonArray.put(timeAndDate);
-		
-		SusiThought result = new SusiThought();
-		result.setData(jsonArray);
-		return result;
-	}
+    private static final long serialVersionUID = 6808423132726076271L;
+
+    @Override
+    public String getAPIPath() {
+        return "/api/timeanddate.json";
+    }
+
+    @Override
+    public BaseUserRole getMinimalBaseUserRole() {
+        return BaseUserRole.ANONYMOUS;
+    }
+
+    @Override
+    public JSONObject getDefaultPermissions(BaseUserRole baseUserRole) {
+        return null;
+    }
+
+    public JSONObject serviceImpl(
+            Query call,
+            HttpServletResponse response,
+            Authorization rights,
+            JSONObjectWithDefault permissions
+    ) throws APIException {
+        return timeAndDate();
+    }
+
+    public static SusiThought timeAndDate() {
+
+        JSONObject timeAndDate = new JSONObject();
+
+        Date time_and_date = new Date();
+        String time_and_date_UTC = timeToUTC(time_and_date).toString().replaceAll(
+                time_and_date.toString().substring(20,23),
+                "UTC"
+        );
+
+        timeAndDate.put("local_time_and_date", time_and_date.toString());
+        timeAndDate.put("UTC_time_and_date", time_and_date_UTC);
+        timeAndDate.put("time_diff", timeDiff(time_and_date));
+
+        JSONArray jsonArray = new JSONArray();
+        jsonArray.put(timeAndDate);
+
+        SusiThought result = new SusiThought();
+        result.setData(jsonArray);
+        return result;
+    }
+
+    private static Date timeToUTC(Date date) {
+
+        SimpleDateFormat dateFormatGmt = new SimpleDateFormat("yyyy-MMM-dd HH:mm:ss");
+        dateFormatGmt.setTimeZone(TimeZone.getTimeZone("GMT"));
+
+        //Local time zone   
+        SimpleDateFormat dateFormatLocal = new SimpleDateFormat("yyyy-MMM-dd HH:mm:ss");
+
+        try {
+           //Time in GMT
+           date = dateFormatLocal.parse(dateFormatGmt.format(date));
```

Comment:
The question is that you reassign param `date` in the method. To avoid this, you can assign date to a local variable and then return it. see [https://stackoverflow.com/a/13507090/5767950](https://stackoverflow.com/a/13507090/5767950)